src/astra_driver.cpp

```diff
@@ -175,8 +175,14 @@ bool AstraDriver::EnableStreaming(std_srvs::SetBool::Request &req, std_srvs::Set
   if (req.data && !enable_streaming_) // Enable streaming
   {
     if (device_ && !device_->isDepthStreamStarted()) device_->startDepthStream();
-    if (device_ && !device_->isIRStreamStarted()) device_->startIRStream();
-    if (device_ && !device_->isColorStreamStarted()) device_->startColorStream();
+    if (!rgb_preferred_)
```

Comment:
Will it be possible to stream IR if rgb_preferred is true, or stream color if rgb_preferred is false?